dom.bs

```diff
@@ -1204,6 +1220,9 @@ for discussion).
  <li><p>Append (<var>target</var>, <var>targetOverride</var>, <var>relatedTarget</var>) to <var>event</var>'s
  <a for=Event>path</a>.
 
+ <li><p>Let <var>activationTarget</var> be <var>target</var>, if <var>target</var> has
```

Comment:
No comma before if. ("Let relatedTarget be the result of retargeting event’s relatedTarget against target if event’s relatedTarget is non-null, and null otherwise.")